CHANGELOG.md

```diff
@@ -13,13 +13,18 @@ The format is based on [Keep a Changelog].
   CTRLF, because (for some reason) it uses `sit-for` to display its
   blink and this of course causes search results to be delayed
   ([#45]).
+* More Isearch compatibility was added in [#48]. Additions include:
+  `ctrlf-occur`, `ctrlf-forward-symbol`,
+  `ctrlf-forward-symbol-at-point`, `ctrlf-toggle-regexp`,
+  `ctrlf-toggle-symbol`, and matching Isearch bindings for those.
```

Comment:
Perhaps this would work better in the Features section (can be added just above Enhancements)?